src/pages/user/components/LoadingUserList.tsx

```diff
@@ -0,0 +1,14 @@
+import { Box, SimpleGrid, SkeletonCircle, SkeletonText } from '@chakra-ui/react'
+
+export const LoadingUserList = ({ count }: { count: number }) => {
+  return (
+    <SimpleGrid columns={{ sm: 1, lg: 2 }} gap={4}>
+      {[...Array(count)].map((_e, i) => (
```

Comment:
Array(count).fill().map... would be more readable, but it works and I learned a new thing about empty arrays 👯‍♀️